packages/react/src/textarea/Textarea.css.ts

```diff
@@ -0,0 +1,80 @@
+import { theme } from "../styles";
+import { style } from "../vanilla-extract";
+import { recipe } from "../vanilla-extract";
```

Comment:
We can import it in the same line

```suggestion
import { recipe, style } from "../vanilla-extract";
```